site/src/components/Screens/SendLoginCodeScreen.tsx

```diff
@@ -90,6 +105,7 @@ export const SendLoginCodeScreen: VFC<SendLoginCodeScreenProps> = ({
       </Typography>
       <Box width="100%" component="form" onSubmit={handleSubmit} noValidate>
         <TextField
+          sx={{ marginBottom: 2 }}
           required
           type="email"
           onBlur={() => setTouchedEmailInput(true)}
```

Comment:
It's not ideal that users who want to sign up will see an error message on this input when they blur it to click the 'Create an account' link - maybe we can set the condition for showing 'empty email' errors to be clicking on the 'log in' button? Or just rely on the native browser 'required' handling?

Reply:
The native `required` handling would introduce popups to the UI which I believe in this context are unwanted. Will change this to be called when clicking the `log in` button.